ccmlib/node.py

```diff
@@ -1311,7 +1311,22 @@ def stress_process(self, stress_options, **kwargs):
                 if not [opt for opt in stress_options if opt.startswith('jmx=')]:
                     stress_options.extend(['-port', 'jmx=' + self.jmx_port])
 
+        if "-schema" in stress_options and "replication" in stress_options[i := stress_options.index("-schema") + 1]:
```

Comment:
would it work with this existing code:
```
node1.stress(stress_options=["write", "cl=ONE", "n=10000", "-schema replication(factor=1)", "-mode cql3 native", "-rate threads=10"], capture_output=True)
```
?

Reply:
hmm, it expects the replication option to be in the following element, but I guess we make it work this way as well